devito/finite_differences/coefficients.py

```diff
@@ -130,7 +130,7 @@ class Substitutions(object):
     Now create a Devito equation and pass to it 'subs'
 
     >>> from devito import Eq
-    >>> Eq(u.dt+u.dx, coefficients=subs)
+    >>> Eq(u.dt+u.dx, coefficients=subs).evaluate
```

Comment:
why adding evaluate here?

Reply:
To show the custom coefficient are indeed used